tests/always/test_providers_manager.py

```diff
@@ -384,6 +384,11 @@ def test_notification(self):
         notification_class_names = list(provider_manager.notification)
         assert len(notification_class_names) > 5
 
+    def test_auth_managers(self):
+        provider_manager = ProvidersManager()
+        notification_class_names = list(provider_manager.auth_managers)
+        assert len(notification_class_names) > 0
```

Comment:
Should we check the names contain the expected content?

Reply:
We could - but this would be merely a convention. I think this test is mostly there to test that the command does not crash and produces "some" responses.  So it's quite ok to leve it as-is.